MarketMatrix/src/components/DiscoverScreen/AssetItem.tsx

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable react-native/no-inline-styles */
+import React from 'react';
+import {View, Text, StyleSheet, Image} from 'react-native';
+import {Stock} from '../../constants/Interfaces';
+import {Colors} from '../../constants/Colors';
+import CardContainer from '../UI/CardContainer';
+
+interface AssetItemProps {
+  stock: Stock;
+}
+
+const AssetItem = ({stock}: AssetItemProps) => {
+  const change = stock.priceChangePercentage;
+  return (
+    <CardContainer>
+      <View style={styles.container}>
+        <View style={styles.leftContainer}>
+          <View style={styles.titleContainer}>
+            <Image source={{uri: stock.image}} style={styles.image} />
+            <View>
+              <Text style={styles.ticker}>{stock.ticker}</Text>
+              <Text style={styles.name}>{stock.companyName}</Text>
+            </View>
+          </View>
+        </View>
+
+        <View style={styles.rightContainer}>
+          <Text style={styles.price}>{stock.price}</Text>
+
+          <Text
+            style={{
+              color: change > 0 ? Colors.green : Colors.pink,
+              fontFamily: 'monospace',
+              fontSize: 14,
+            }}>
+            {change > 0 ? '+' : ''}
+            {change.toFixed(2)}%
+          </Text>
+        </View>
+      </View>
+    </CardContainer>
+  );
+};
+
+export default AssetItem;
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+  },
+  leftContainer: {
+    flex: 1,
+    gap: 5,
+  },
+  rightContainer: {
+    alignItems: 'flex-end',
+    gap: 6,
+  },
+
```

Comment:
There are lines with spaces in styles, maybe you should delete them for consistency